packages/edit-site/src/components/sidebar-navigation-screen-page/index.js

```diff
@@ -18,14 +29,124 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import { unlock } from '../../private-apis';
 import { store as editSiteStore } from '../../store';
 import SidebarButton from '../sidebar-button';
+import SidebarNavigationSubtitle from '../sidebar-navigation-subtitle';
+import SidebarDetails from '../sidebar-navigation-data-list';
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+const POST_STATUS_LABELS = {
+	publish: __( 'Published' ),
+	future: __( 'Scheduled' ),
+	draft: __( 'Draft' ),
+	pending: __( 'Pending' ),
+};
+
+function getPageDetails( page ) {
+	if ( ! page ) {
+		return [];
+	}
+	const details = [
+		{
+			label: 'Parent',
+			value: page?.parentTitle,
+		},
+	];
+
+	if ( page?.templateTitle ) {
+		details.push( {
+			label: 'Template',
+			value: page.templateTitle,
+		} );
+	}
+
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = page?.content?.raw
+		? wordCount( page.content.raw, wordCountType )
+		: 0;
+	const readingTime = Math.round( wordsCounted / AVERAGE_READING_RATE );
```

Comment:
How useful are wordcount and reading time stats to user in this context?

Reply:
Good question — I quite like the inclusion, it could be useful if you're wanting to see at a glance how long a really long page might be, but I imagine it might be more useful for posts?